### DISCUSSION TITLE
[draft-v1] Section 2.2.1: Add reference to wcslib paper?

### DISCUSSION THREAD

**Discussion by pllim** [astropy/astropy-v5.0-paper#52] [Text comments] [answered]
> \astropysubpkg{wcs} sub-package allowed access to WCS information provided in, e.g., FITS files

I think this was referring to the FITS WCS that wraps `wcslib`, so we should cite the main reference for https://www.atnf.csiro.au/people/mcalabre/WCS/ ? I don't know which paper is the one to cite. @mcara, do you know?

**Comment by mcara** [comment]
I do not think this sentence, as formulated, refers to WCSLIB itself but rather to `wcs`' sub-package capabilities. That's how I understand (==read) it. However, depending on what you think that should be cited, I would suggest:

WCSLIB: https://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/index.html
FITS WCS: https://fits.gsfc.nasa.gov/fits_wcs.html 

The last link lists references to the main papers describing FITS WCS standard (Papers I, II and *maybe* III). If citing specific things from the FITS WCS I would cite the specific paper that provided the information. But Paper I lays the foundations of the FITS WCS and so if I had to choose one paper - I would choose this one: ["Representations of world coordinates in FITS"](https://www.aanda.org/articles/aa/pdf/2002/45/aah3859.pdf).

[Also](https://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/software.html), WCSLIB (software library) is included in the Astrophysics Source Code Library (ASCL https://ascl.net/) as record ascl:1108.003 (https://ascl.net/1108.003), and in the Astrophysics Data System (ADS https://ui.adsabs.harvard.edu/) with bibcode 2011ascl.soft08003C (https://ui.adsabs.harvard.edu/abs/2011ascl.soft08003C).

Maybe @nden could provide better advice.

**Reply by nden** [reply]
I agree this is the astropy.wcs subpackage. However, if you need a reference to FITS WCS the second link @mcara mentioned is the one to use. `wcslib` implements the 5 main papers listed on that site. I'm not aware of a citation for wcslib.

**Reply by perrygreenfield** [reply]
5 papers? I count only 4. There is that ghost paper that died before publication...

wcslib does implement the ghost paper though.

**Comment by pllim** [comment, marked as answer]
After discussing with @mcara and @nden offline, we agree no new citation is necessary. We will close this discussion without further action. Thanks, everyone!
